index.css

```diff
@@ -411,7 +520,8 @@ li {
 .project-description {
   margin-top: 13px;
   width: 95%;
-  max-width: 640px;
+
+  /* max-width: 640px; */
```

Comment:
[OPTIONAL] Unfortunately, the previous reviewer's comment for the full-screen modal was more like a personal idea. We need to do what we see on the Figma template. So you can go back to the previous width for the modal if you would like to 👍